src/main.scss

```diff
@@ -0,0 +1,27 @@
+@import '~@fullcalendar/core/main.css';
+@import '~@fullcalendar/daygrid/main.css';
+
+.container-app {
+  font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+.container-app-top {
+  margin: 0 0 3em;
+}
+
+.calendar {
+  margin: 0 auto;
+  max-width: 900px;
+}
+.calendar .fc-day-header {
```

Comment:
Note that `stylelint` is missing from lint step (also need to make sure CircleCI is configured).